app/requirement_improver.py

```diff
@@ -116,35 +115,55 @@ def whole_phrase_regexp(phrase):
             phrase = phrase.replace(' ', '\s')
 
             try:
-                return re.compile(r'\b{0}\b'.format(phrase), flags=re.I|re.X)
+                return re.compile(r'\b{0}\b'.format(phrase), flags=re.I | re.X)
             except re.error:
-                return re.compile(r'\b\{0}\b'.format(phrase), flags=re.I|re.X)
+                return re.compile(r'\b\{0}\b'.format(phrase), flags=re.I | re.X)
+
+        def is_userstory(txt):
+            """ checks weather given sentence confirms to user-story template"""
+            x = re.findall("^As [an|a]", txt)
+            y = re.findall(", [Ii] want to", txt)
+            if (x) and (y):
+                return True
+            else:
+                return False
+
+        def is_self_pronoun(txt):
+            """ checks weather given word is of any first-person-form"""
+            x = re.matc("i|my|mine|myselfe|me", txt)
```

Comment:
Hi and thanks for your contribution.
Please, double-check the correct method name
`re.match()`